### PR TITLE
[Config] Fix checking class existence freshness

### PR DESCRIPTION
**Pull request by nicolas-grekas**
| Q             | A
| ------------- | ---
| Branch?       | 3.3
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #23753
| License       | MIT
| Doc PR        | -

### CODE REVIEW

**Review by chalasr — APPROVED** [review]

**Comment by gharlan** [comment]
The fix works for me.

**Review by fabpot — APPROVED** [review]

**Review by ogizanagi — APPROVED** [review]

**Review by Simperfit — APPROVED** [review]

File: src/Symfony/Component/Config/Tests/Resource/ClassExistenceResourceTest.php [hunk 1]
```diff
@@ -74,6 +75,22 @@ public function testExistsKo()
         }
     }
 
+    public function testBarParentWithTimestamp()
```
Comment (cH40z-Lord) [inline src/Symfony/Component/Config/Tests/Resource/ClassExistenceResourceTest.php]: Shouldn't `Bar` be `Bad` in regards to the class name itself, `BadParent`?

Same goes for line 88.

Comment (nicolas-grekas) [inline src/Symfony/Component/Config/Tests/Resource/ClassExistenceResourceTest.php]: good catch, fixed

**Review by javiereguiluz — APPROVED** [review]

**Comment by ogizanagi** [comment]
Thank you @nicolas-grekas.

**Comment by Simperfit** [comment]
@nicolas-grekas @fabpot Can we tag a new release with this fix ? it seems important to me.

**Pull request merged**